File: problems/integer-to-roman/boilerplate-full/function.py
#USER_CODE_HERE
import sys
from typing import List

# Set recursion limit higher for recursive problems
sys.setrecursionlimit(2000)

if __name__ == "__main__":
    # Input reading for the specific problem
    num = int(input())

    sol = Solution()
    
    # Call the solution function
    result = sol.intToRoman(num)
    
    # Print the result
    print(result)

File: problems/letter-combinations-of-a-phone-number/boilerplate-full/function.java
//USER_CODE_HERE
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String digits = sc.next();
        Solution sol = new Solution();
        List<String> result = sol.letterCombinations(digits);
        System.out.println(result);
    }
}